VIS/Areas/VIS/Scripts/app/Framework/amountdivision.js

```diff
@@ -223,10 +227,18 @@
             divAcctSchemaCtrlWrp.append(cmbAcctSchema).append(lblAcctSchema);
             var divDimType = $("<div class='VIS-AMTD-formData input-group vis-input-wrap'>");
             var divDimTypeCtrlWrp = $("<div class='vis-control-wrap'>");
-            var divButton2 = $('<div class="vis-excelimportbtn VIS-AMTD-formBtns" style=" display: none; "><a tabindex="" class="vis vis-doc-excel"></a></div>');
+            var divButton2 = $('<div for="VA028-file-input" class="vis-excelimportbtn VIS-AMTD-formBtns" style=" display: block; "><a tabindex="" class="vis vis-doc-excel"></a></div>');
             divDimType.append(divDimTypeCtrlWrp);
             divDimType.append(divButton2);
             divDimTypeCtrlWrp.append(cmbDimensionType).append(lblDimensionType);
+
+            /* Create the element of input File */
+            var divExcelDim = $('<div style="float: right;margin-right: 5px;display:none" ><input id="VA028-file-input" style="display:none" type="file"  accept=".xls,.xlsx"><label title="'
+                + VIS.Msg.getMsg("VA028_ExcelImport") + '"  for="VA028-file-input" class="VA028-change-picture-ico"></label></div>');
```

Comment:
replace VA028 with VIS-AMTD- which is our standard notation for amount dimension control